addons/coupon/views/coupon_program_views.xml

```diff
@@ -188,25 +183,24 @@
         </field>
     </record>
 
-    <menuitem id="menu_coupon_type_config" action="sale_coupon_program_action_coupon_program" parent="sale.product_menu_catalog" name="Coupon Programs" groups="sales_team.group_sale_manager" sequence="5"/>
-
-
     <!-- Promotion Program -->
 
-    <record id="sale_coupon_program_view_promo_program_form" model="ir.ui.view">
-        <field name="name">sale.coupon.promotion.program.form</field>
-        <field name="model">sale.coupon.program</field>
-        <field name="inherit_id" ref="sale_coupon_program_view_form_common"/>
+    <record id="coupon_program_view_promo_program_form" model="ir.ui.view">
+        <field name="name">coupon.promotion.program.form</field>
+        <field name="model">coupon.program</field>
+        <field name="inherit_id" ref="coupon_program_view_form_common"/>
         <field name="mode">primary</field>
         <field name="arch" type="xml">
             <xpath expr="//div[@name='title']" position="inside">
                 <label class="oe_edit_only" for="name" string="Promotion Program Name"/>
                 <h1><field name="name" class="oe_title" placeholder="Promotion Program Name..." height="20px"/></h1>
             </xpath>
-            <xpath expr="//field[@name='order_count']/.." position="before">
-                <button class="oe_stat_button" type="action" icon="fa-ticket" name="%(sale_coupon.sale_coupon_action)d" attrs="{'invisible': [('promo_applicability', '=', 'on_current_order')]}">
-                    <field name="coupon_count" string="Coupons" widget="statinfo"/>
-                </button>
+            <xpath expr="//widget[@name='web_ribbon']" position="before">
```

Comment:
since you're probably going to add buttons in the button box, it will probably be easier to have the button box in the base view in the coupon module instead of adding it here

Reply:
fyi, it isn't displayed if there's no button in it so it doesn't make a weird layout if neither sale_coupon nor pos_coupon is installed